benchmarks/src/main/java/org/apache/druid/benchmark/LongCompressionBenchmarkFileGenerator.java

```diff
@@ -42,6 +42,7 @@
 import java.util.List;
 import java.util.Map;
 
+@SuppressWarnings("SSBasedInspection") // static field(s) cannot be final because set in an static method
```

Comment:
If the suppression is done for a single field (`dirPath`), it should be on the field still.